frontend/src/components/common/table/styles.module.scss

```diff
@@ -30,6 +30,14 @@
   padding: 20px;
 }
 
+.tableBodyRowData:first-child {
+  width: 2%;
+}
+
+.tableBodyRowData:last-child {
```

Comment:
But this will also affect the last row of tables, which don't have "Actions" cell